workflow-dev/tpl/storage.sh

```diff
@@ -1,9 +1,7 @@
 #!/bin/bash
 store=`grep "storage\s*=\s*" $HELM_GENERATE_DIR/tpl/generate_params.toml | cut -d "=" -f2 | tr -dc '[:alnum:]\n\r'`
-if [ -n "$STORAGE_TYPE" ]; then
-  if [ "$STORAGE_TYPE" != "minio" ]; then
-    rm -rf $HELM_GENERATE_DIR/manifests/deis-minio-*
-  fi
+if [ -n "$STORAGE_TYPE" ] && [ "$STORAGE_TYPE" != "minio" ]; then
+  rm -rf $HELM_GENERATE_DIR/manifests/deis-minio-*
 elif [ "$store" != "minio" ]; then
   rm -rf $HELM_GENERATE_DIR/manifests/deis-minio-*
 else
```

Comment:
I understand what you are trying to say but we can't move away from negative logic as the default value present in the toml file is minio.